zebra-chain/src/sapling/spend.rs

```diff
@@ -199,7 +199,7 @@ impl ZcashDeserialize for Spend<PerSpendAnchor> {
             // See [`commitment::NotSmallOrderValueCommitment::zcash_deserialize`].
             cv: commitment::NotSmallOrderValueCommitment::zcash_deserialize(&mut reader)?,
             // Type is `B^{[ℓ_{Sapling}_{Merkle}]}`, i.e. 32 bytes
-            per_spend_anchor: tree::Root(reader.read_32_bytes()?),
+            per_spend_anchor: tree::Root::try_from(reader.read_32_bytes()?)?,
```

Comment:
Optional: This might also work:

```suggestion
            per_spend_anchor: (&mut reader).zcash_deserialize_into()?,
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/3544/commits/6a8b4b8a18f4206da6e712cf6fc31d8ee5a55c78